_docs_integrate/data-model-overview.md

```diff
@@ -477,7 +487,9 @@ RequestItems can be sent inside of a [Request](#request) and specify what should
 | mustBeAccepted        | `boolean`                | The mandatory `mustBeAccepted` property is used to differentiate between required and optional RequestItems within the Request.<br>{::nomarkdown}<ul><li>If <code>mustBeAccepted</code> is set to <code>true</code>, the peer cannot <a href="https://enmeshed.eu/use-case-consumption-accept-incoming-request">accept the Request</a> without accepting this RequestItem.</li><li>If <code>mustBeAccepted</code> is set to <code>false</code>, the peer can <a href="https://enmeshed.eu/use-case-consumption-accept-incoming-request">accept the Request</a> even though the RequestItem is rejected.</ul>{:/} |
 | requireManualDecision | `boolean` \| `undefined` | To block the automated acceptance of Requests, the requireManualDecision property can be set to true. The default is, that each RequestItem may be automatically processed on the peer side. If the sender would like to have an enforced manual acceptance step the requireManualDecision property can be set to true.                                                                                                                                                                                                                                                                                          |
 
-#### [ProposeAttributeRequestItem]({% link _docs_integrate/request-and-response-introduction.md %}#proposeattributerequestitem)
+#### ProposeAttributeRequestItem
+
+For more information you should checkout the section [ProposeAttributeRequestItem]({% link _docs_integrate/request-and-response-introduction.md %}#proposeattributerequestitem) of the [Request and Response introduction]({% link _docs_integrate/request-and-response-introduction.md %}). A brief overview of the various operations which can be performed with Attributes can also be found in the section [Attribute management options]({% link _docs_integrate/attribute-introduction.md %}#attribute-management-options).
```

Comment:
Don't link to the Attribute management options here (an overview is also found in the Request and Response Introduction), but instead to `propose-attributes-to-peer`.